### ISSUE TITLE
Issue with app trying to send runtime error email

### ISSUE THREAD

**Issue by dccomic** [toopriddy/mytime#452]
For some reason MyTime keeps trying to send an email to support regarding an issue with a runtime error. It pops up each time I open MyTime. I finally decided to just click send in case that would be helpful.
Thanks,
Don

**Comment by Appaljack** [comment]
The app has not been updated in years so you might try installing and re-installing. It basically still works but not flawlessly and does not have the current publications. It is fine as an rv book.


Gary W Slocum423-900-4046 call/text

On Sunday, September 29, 2024, 9:12 PM, dccomic ***@***.***> wrote:




For some reason MyTime keeps trying to send an email to support regarding an issue with a runtime error. It pops up each time I open MyTime. I finally decided to just click send in case that would be helpful.
Thanks,
Don

**Comment by dccomic** [comment]
Hi Gary,
Thanks for letting me know, I had no idea the app was not being supported.
I have enjoyed using it so it is going to be a big loss for me and others
using it.
Kind regards,
Don

**Comment by Appaljack** [comment]
We use the address section for calls and handwrite everything in the comment area as reporting requirements are all different as well. Support died before Covid but that 3.5 years all we did was write so not needed. Our Hall uses NW Publisher for the Congregation part of things. Not sure if everything still works in iOS 18.0 but seems that the address part is stable and you can still email a call if needed. Not sure if other apps exist other than using Apple Notes or something.


Gary W Slocum423-900-4046 call/text

On Monday, September 30, 2024, 10:32 AM, dccomic ***@***.***> wrote:



Hi Gary,
Thanks for letting me know, I had no idea the app was not being supported.
I have enjoyed using it so it is going to be a big loss for me and others
using it.
Kind regards,
Don

**Comment by dccomic** [comment]
Yes, I still use MyTime for RV’s etc, but I want something that works
similar but that will get updates moving forward, because it needs to be
completely reliable due to the importance keeping track of our calls.
I’m going to try using the “Witness Work” app which is also a donation
based app. It seems pretty robust so I feel confident in using it to keep
track of my calls and time.
Our congregation uses NWP as well, and there are a number of other popular
apps that congregations use. NWP has some flaws, but overall, it is
wonderful and has certainly made things easier for the friends. Hopefully
the branch will eventually provide something similar so that all the
congregations are uniform and organized.
Thanks for the info my brother!

**Comment by Appaljack** [comment]
Will look up Witness Work, thanks for the tip!


Gary W Slocum423-900-4046 call/text

On Monday, September 30, 2024, 11:05 AM, dccomic ***@***.***> wrote:



Yes, I still use MyTime for RV’s etc, but I want something that works
similar but that will get updates moving forward, because it needs to be
completely reliable due to the importance keeping track of our calls.
I’m going to try using the “Witness Work” app which is also a donation
based app. It seems pretty robust so I feel confident in using it to keep
track of my calls and time.
Our congregation uses NWP as well, and there are a number of other popular
apps that congregations use. NWP has some flaws, but overall, it is
wonderful and has certainly made things easier for the friends. Hopefully
the branch will eventually provide something similar so that all the
congregations are uniform and organized.
Thanks for the info my brother!

**Comment by dccomic** [comment]
You’re welcome Gary. Take care!

**Comment by toopriddy** [comment]
there was an update to MyTime 3.11.0 which fixes this and is the first in a long time because of lots of reasons. I hope that now that I have an AI buddy helping me that things will be a lot more stable
